myvar.c

```diff
@@ -85,18 +85,26 @@ const char *myvar_get(const char *var)
 
 /**
  * myvar_set - Set the value of a "my_" variable
- * @param var Variable name
- * @param val Value to set
+ * @param var    Variable name
+ * @param val    Value to set
+ * @param append True to append to an existing value, if any
  */
-void myvar_set(const char *var, const char *val)
+void myvar_set(const char *var, const char *val, bool append)
```

Comment:
I would avoid boolean parameters, and have

```C
void myvar_set(const char *var, const char *val);
void myvar_append(const char *var, const char *val);
```

perhaps both implemented in terms of a
```C
static struct MyVar *myvar_find(const char *var);
```

Reply:
Sure, will do in v2.